spec/models/ems_refresh/refreshers/azure_refresher_spec.rb

```diff
@@ -14,6 +14,8 @@
   end
 
   it "will perform a full refresh" do
+    pending # This test fails sporadically
```

Comment:
Looks good @bzwei , we don't need the comment though.  Instead, pass it as a string to pending it will printed as the reason we're skipping this example.